rs/src/gstd/mod.rs

```diff
@@ -41,3 +41,31 @@ impl ExecContext for GStdExecContext {
         *self.msg_id.get_or_init(msg::id)
     }
 }
+
+pub struct CommandResult<T>(T, ValueUnit);
```

Comment:
What do you think about naming it as `CommandOutput` in order to avoid confusion with the `Result` type

Reply:
renamed to `CommandReply`